src/traces/pie/attributes.js

```diff
@@ -79,7 +79,27 @@ module.exports = {
 
     text: {
         valType: 'data_array',
```

Comment:
@alexcjohnson I just noticed that `text` for pie is declared as a `data_array` whereas other trace types have it as a `string` with `arrayOk: true`. Was that on purpose or should we allow single-value `text` input for pie traces?